testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -678,15 +682,29 @@ impl SubmissionWorker {
             .iter_mut()
             .choose_multiple(&mut rng, batch_size);
         let mut requests = Vec::with_capacity(accounts.len());
+        let mut invalid_size = if self.invalid_tx != 0 {
+            // if enable mix invalid tx, at least 1 invalid tx per batch
+            max(1, accounts.len() * self.invalid_tx as usize / 100)
+        } else {
+            0
+        };
         for sender in accounts {
             let receiver = self
                 .all_addresses
                 .choose(&mut rng)
                 .expect("all_addresses can't be empty");
+            let chain_id = if invalid_size > 0 {
+                // we use 255 to be a fake chain id for invalid tx
+                invalid_size -= 1;
+                ChainId::new(255)
+            } else {
+                self.chain_id
+            };
             let request =
-                gen_transfer_txn_request(sender, receiver, SEND_AMOUNT, self.chain_id, gas_price);
+                gen_transfer_txn_request(sender, receiver, SEND_AMOUNT, chain_id, gas_price);
```

Comment:
While invalid ChainID works as invalid TXN, but it's not the full picture.  

Can you cover other edge cases of invalid TXNs?  For example, mixing any of the following, gas fees that are out of range, none existent sender and/or receiver accounts on chain, mint.